apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -292,6 +294,8 @@ export const onSaveComplete =
     dispatch(setSavedAiCustomizations(currentAiCustomizations));
     // Notify the UI that the save is complete.
     dispatch(endSave());
+    // Send a report that user has started the aichat level - a teacher will view that level is now in progress.
```

Comment:
nit: `Send a report that the user has started the aichat level after a successful save - ...`?